lib/galaxy/datatypes/tabular.py

```diff
@@ -15,6 +15,9 @@
 from cgi import escape
 from json import dumps
 
+import pysam
+import pysam.bcftools
```

Comment:
This import seems unused.

Reply:
No, that's needed because this isn't imported by default:
```
In [1]: import pysam

In [2]: pysam.bcftools
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-cb47258bfbc9> in <module>()
----> 1 pysam.bcftools

AttributeError: 'module' object has no attribute 'bcftools'

In [3]: import pysam.bcftools

In [4]: pysam.bcftools
Out[4]: <module 'pysam.bcftools' from '/Users/mvandenb/.venv/lib/python2.7/site-packages/pysam/bcftools.pyc'>
```

Reply:
I mean that `pysam.bcftools` is used in binary.py , but not in this file.

Reply:
oh you're right, I initially used this to replace `bcftools concat` but that didn't work!

Reply:
No worries, thanks for fixing it! There are 3 small review comments left you may have missed, then it should be ready to merge.

Reply:
Forgot to add them. Thanks again.